services/node_stake_update.proto

```diff
@@ -37,10 +37,50 @@ message NodeStakeUpdateTransactionBody {
    */
   Timestamp end_of_staking_period = 1;
 
+  /**
+   * The maximum reward rate, in tinybars per whole hbar, that any account can receive in a day.
+   */
+  int64 max_daily_reward_rate = 2;
+
+  /**
+   * The fraction of the network and service fees (as a whole number in the range [0, 100]) paid to 
+   * the node reward account 0.0.801.
+   */
+  int32 node_reward_fees_percent = 3;
+
   /**
    * Staking info of each node at the beginning of the new staking period
    */
-  repeated NodeStake node_stake = 2;
+  repeated NodeStake node_stake = 4;
```

Comment:
believe we can't have breaking changes this time, so existing fields number should stay the same:

```suggestion
  repeated NodeStake node_stake = 2;
```